Makefile

```diff
@@ -4,29 +4,31 @@ GOBIN = go
 # i.e. ignore local $GOPATH/src installed sources
 GOPATH_TMP = $(CURDIR)/.GOPATH
 GO = /usr/bin/env GOPATH=$(GOPATH_TMP) $(GOBIN)
-GO_FLAGS =
 GOFMT = gofmt
 VERSION = dev-$(shell date +%FT%T%z)
 
-OS = linux
-ARCH = amd64
+GO_OS = GOOS="${GOOS}"
+GO_ARCH = GOARCH="${GOARCH}"
 BINARY = kubeapps
 GO_PACKAGES = $(IMPORT_PATH)/cmd $(IMPORT_PATH)/pkg/...
 GO_FILES := $(shell find $(shell $(GOBIN) list -f '{{.Dir}}' $(GO_PACKAGES)) -name \*.go)
-GO_FLAGS = -ldflags="-w -X github.com/kubeapps/kubeapps/cmd.VERSION=${VERSION}"
+GO_FLAGS = -ldflags='-extldflags "-static" -w -X github.com/kubeapps/kubeapps/cmd.VERSION=${VERSION}'
+GO_LDFLAGS =
```

Comment:
is `GO_LDFLAGS` used anywhere?

Reply:
It's used nowhere. It's actually been there for a while so I just don't touch it. Will remove it now